api/src/main/resources/workbench.yaml

```diff
@@ -647,12 +647,12 @@ paths:
       description: Quick search for criteria that matches on code and name.
       operationId: "getCriteriaTreeQuickSearch"
       parameters:
-        - in: path
+        - in: query
```

Comment:
I think that we want this to be path? So we can call the api like so: http://all-of-us-workbench-test.appspot.com/api/v1/cohortbuilder/quicksearch/ICD9/cho ... Also, will we have a min characters before we calls this api?

Reply:
The API call is not made by the frontend until there are three or more chars in the search string (that check is [here](https://github.com/all-of-us/workbench/pull/289/files#diff-caa6588446cbd142d58bd51a9b342050R67))

I think these should be query parameters.  But if they should be path parameters the path needs to be altered to reflect that.

Reply:
the api is implemented to use path.